### ISSUE TITLE
fix: responsive dashboard cards view

### ISSUE THREAD

**Issue by matt-d-webb** [chess-centre/platform#543]
Currently in some default layouts the responsible view is broke!

![image](https://user-images.githubusercontent.com/36933715/149635982-147556e7-dcf8-4c86-afb0-dfde30265800.png)

The above is just an example, these "Stats" need to be refactored as they're currently poorly constructed:

```jsx
    <div className="relative text-center xl:text-left bg-white dark:bg-gray-800 px-4 xl:pb-12 xl:pt-6 xl:px-6 shadow rounded-lg overflow-hidden">
      <div>
        <div className="xl:absolute p-2">
          <span className="text-teal-500 text-3xl">
            <i className="fad fa-chess"></i>
          </span>
        </div>
        <p className="xl:ml-16 text-md font-medium text-gray-500 dark:text-gray-300 truncate">
          {title}
        </p>
      </div>
      <div className="text-center xl:ml-16 xl:flex xl:items-baseline  pb-7 xl:pb-0">
        <div className="text-5xl font-bold text-gray-900 dark:text-white m-auto xl:m-0">
          {total && (
            <Link className="" to="/app/games">
              {total}{" "}
            </Link>
          )}
        </div>

        <div className="relative m-auto xl:m-0 flex flex-none w-20 text-center items-center px-2 py-0.5 rounded text-xs font-medium bg-teal-100 text-teal-700 mb-3 xl:ml-2 mt-6 xl:mr-2">
          Standard
          <span className="absolute text-teal-500 ml-2 right-2">
            {standard}
          </span>
        </div>

        <div className="relative m-auto xl:m-0 flex flex-none w-20 text-center items-center px-2 py-0.5 rounded text-xs font-medium bg-orange-100 text-orange-700">
          Rapid
          <span className="absolute text-orange-700 ml-2 text-right right-2">
            {rapid}
          </span>
        </div>
      </div>

      <div className="absolute bottom-0 inset-x-0 bg-gray-50 dark:bg-gray-800 px-4 py-2 xl:px-6 border-t border-gray-100"></div>
    </div>
 ```

Reference code: https://github.com/chess-centre/platform/blob/master/chess-centre-app/src/components/OverviewStats/Stats.js#L56

There is obviously a much cleaner way to construct these which will avoid such issue arising.

These should be tested against a greater number of:

- Games > 1,000
- Events > 1,000
- Ratings = 3 (standard, rapid, bullet)

**Comment by arpit-maurya** [comment]
Hi, matt-d-webb ,i am Arpit Maurya (Front-end Developer) i like to contribute in your project

**Comment by matt-d-webb** [comment]
Hey @arpit-maurya! Fantastic 🎉 

I have extracted one of the blocks into the `tailwindcss` playground here: https://play.tailwindcss.com/2jkb7ZzKmd

_Note: I removed the `React` specific aspects._

So, two options, you could simply use this tool to propose the changes for this specific ticket which I'll update into the frontend, or you can fork this project and I can guide you through setting up your dev environment so you can raise the necessary PR(s).

Let me know which you'd prefer. The second option will take around 30-40 minutes to complete, you can make a head start here: https://github.com/chess-centre/platform#prerequisites

**Comment by arpit-maurya** [comment]
hi @matt-d-webb ,thank you for inviting , please  Can you help me to setup project

**Comment by matt-d-webb** [comment]
@arpit-maurya hello there 👋🏼  - apologies for the delay.

If you could drop me a quick email at matt[at]chesscentre.online I will send across the IAM contributor details with quick instructions for pulling a backend dev environment.

**Comment by matt-d-webb** [comment]
@all-contributors add @arpit-maurya for code

**Comment by matt-d-webb** [comment]
Fixed by [#629](https://github.com/chess-centre/platform/pull/629)

<img width="1021" alt="Screenshot 2022-04-24 at 02 11 43" src="https://user-images.githubusercontent.com/36933715/164951462-e65ab083-24ea-4de1-ac42-b73449126211.png">
